sdk/cognitiveservices/azopenai/models.go

```diff
@@ -0,0 +1,209 @@
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is regenerated.
+// DO NOT EDIT.
+
+package azopenai
+
+// ClientGetCompletionsOptions contains the optional parameters for the Client.GetCompletions method.
+type ClientGetCompletionsOptions struct {
+	// placeholder for future optional parameters
+}
+
+// ClientGetEmbeddingsOptions contains the optional parameters for the Client.GetEmbeddings method.
+type ClientGetEmbeddingsOptions struct {
+	// placeholder for future optional parameters
+}
+
+type CompletionRequest struct {
```

Comment:
FYI- all models need a doc comment. Pipeline will fail without one. Might have to add in the autorest file.

Reply:
I added them.